src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -374,6 +374,7 @@ export class VisualizeEmbeddable
         query: this.input.query,
         filters: this.input.filters,
       },
+      searchSessionId: this.input.searchSessionId,
```

Comment:
If we pass in the `searchSessionId` from the visualize app, would that enable sessions for that app too?